### PR TITLE
About app

### PR DESCRIPTION
**Pull request by InfernoCoder**
I have implemented the about button in this PR.

Screenshot - 
screenshot_20161224-144736 1

### CODE REVIEW

File: PowerUp/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -40,14 +40,27 @@
 
         <activity
             android:name=".DressingRoomActivity"
+<<<<<<< HEAD
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/AndroidManifest.xml]: Please remove these merge conflict messages,these won't let the code run properly

File: PowerUp/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -40,14 +40,27 @@
...
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar"/>
+        <activity
+            android:name=".SelectFeaturesActivity"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar"/>
+        <activity
+            android:name=".ScenarioOverActivity"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar"/>
+>>>>>>> upstream/develop
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/AndroidManifest.xml]: Same

**Comment by anubhakushwaha** [comment]
@InfernoCoder Please squash them

**Comment by InfernoCoder** [comment]
@anubhakushwaha  I messed up this PR, so I'm closing this one and adding a new PR soon.

**Comment by anubhakushwaha** [comment]
@InfernoCoder No problem :) Go ahead

**Comment by InfernoCoder** [comment]
Closing this one and adding a new one...

**Pull request closed without merging**